Source/Model/LoadExecutableError.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LoadExecutableError.swift
+//  PPPC Utility
+//
+//  MIT License
+//
+//  Copyright (c) 2019 Jamf Software
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+//
+
+import Foundation
+public enum LoadExecutableError: Error {
+    case identifierNotFound
+    case resourceURLNotFound
+    case codeRequirementError(description: String)
+    case executableNotFound
+    case executableAlreadyExists
+}
+
+extension LoadExecutableError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .identifierNotFound:
+            return "Bundle identifier could not be found."
+        case .resourceURLNotFound:
+            return "Resource URL could not be found."
+        case .codeRequirementError:
```

Comment:
This should pull in the underlying description (the associated string) and add that to the returned string.

```suggestion
        case .codeRequirementError(let underlyingDescription):
```

Reply:
I went back and forth on this one a little bit, which is why I accidentally left the line to pass an associated value without using it.  I think in general more descriptive errors are better. It now looks like this:
<img width="1242" alt="Screen Shot 2020-05-07 at 7 15 44 AM" src="https://user-images.githubusercontent.com/33291843/81294198-26b02a80-9034-11ea-941a-06828d493ffe.png">

The problem is I don't think the underlying error is useful for general users, and for debugging that error is printed to the console. Thoughts?

Reply:
I like the more specific error message.

I think if general users have the error in the UI, they will be able to report a more complete issue here on GitHub.  If it's only in the console, general users will report a bug and our first step will always be to tell them "open the Console.app, try again in PPPC Utility, and then send us the actual underlying error message".